src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -235,19 +225,21 @@
   padding-bottom: var(--pf-c-about-modal-box__close--PaddingBottom);
 
   @media only screen and (min-width: $pf-global--breakpoint--sm) {
+    --pf-c-about-modal-box__close--PaddingRight: var(--pf-c-about-modal-box__close--lg--PaddingRight);
+
     grid-area: 1 / 2;
     justify-content: center; // center the close over the narrow hero image
-    padding-right: 0;
   }
 
   @media only screen and (min-width: $pf-global--breakpoint--lg) {
+    --pf-c-about-modal-box__close--PaddingRight: var(--pf-c-about-modal-box__close--lg--PaddingRight);
+
     justify-content: flex-end; // align flex end the close over the narrow hero image
-    padding-right: var(--pf-c-about-modal-box__close--PaddingRight);
   }
 
   // Close button
   // Since this btn is unique to the about modal, use contextual selectors in this specific instance
-  .pf-c-button.pf-m-plain {
+  .pf-c-button {
     display: flex;
     align-content: center;
     justify-content: center;
```

Comment:
can you assign the border-radius to a global variable ?

Reply:
👍